### PR TITLE
Added deviceupdate-create-account to quickstarts

### PR DESCRIPTION
**Pull request by abatishchev**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Initial commit of first template for Device Update

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@vimeht - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@vimeht - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@vimeht - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@vimeht - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@vimeht - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by abatishchev** [comment]
Test deployment fails with an error:

```
VERBOSE: Performing the operation "Creating Deployment" on target "azdo-712af179-082e-49a7-9dc1-d9de40747709".
New-AzResourceGroupDeployment : 8:50:41 PM - Error: Code=LocationNotAvailableForResourceType; Message=The provided location 'westus' is not available for resource type 'Microsoft.DeviceUpdate/accounts'. List of available regions for the resource type is 'westus2,northeurope,southeastasia'.
```

And that's expected. Our service is only present in West US 2, North Europe, Southeast Asia for now. How can I change the location where it attempts to deploy the template to?

**Comment by azure-quickstarts** [comment]
@vimeht - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@vimeht - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: 101-deviceupdate-create-account/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Create Device Update for IoT Hub account
```
Comment (bmoore-msft) [inline 101-deviceupdate-create-account/README.md:1]: move the sample to the quickstarts/microsoft.deviceupdate folder - remove the 101 from the sample name

File: 101-deviceupdate-create-account/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "accountName": {
+      "type": "string",
```
Comment (bmoore-msft) [inline 101-deviceupdate-create-account/azuredeploy.json:6]: add a defaultValue using uniqueString()

File: 101-deviceupdate-create-account/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "accountName": {
```
Comment (bmoore-msft) [inline 101-deviceupdate-create-account/azuredeploy.parameters.json:5]: remove this when you add the defaultValue

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Pull request merged**